src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitsStorage = new HashMap<>();
+
+    public Map<String, Integer> getFruitsStorage() {
+        return fruitsStorage;
```

Comment:
Exposing the internal storage map directly through `getFruitsStorage` can lead to unintended modifications from outside the class. Consider providing specific methods for adding, removing, or updating entries in the storage to maintain better control over the data.